packages/mobile/src/screens/chat-screen/ChatUserListItem.tsx

```diff
@@ -258,7 +250,15 @@ export const ChatUserListItem = ({ userId }: ChatUserListItemProps) => {
                 )}
               </View>
               {user.does_follow_current_user && canCreateChat ? (
-                <Text style={styles.followsYouTag}>{messages.followsYou}</Text>
+                <Text
+                  fontSize='xxs'
+                  weight='heavy'
+                  color='neutralLight4'
```

Comment:
Worth thinking about going forward if we should add the semantic variants of text color and use those instead (i.e. `textSubdued`)